scheduler/docs/scheduler-rest-api.asc

```diff
@@ -69,6 +69,7 @@ The request body is list of job maps with each entry containing a map of the fol
 |JSON Key | Type | Description
 |`uuid` | string | a user provided UUID for tracking and referring to the job.
 |`command` | string | the command to run
+|`cook_executor` | boolean | Flag to force use of the Cook executor, defaults to false.
```

Comment:
We already have a `custom-executor` flag, following suite. With an `executor` flag the value should be 'cook|custom|mesos', but it is unlikely that we are going to drop `custom-executor` flag.